compiler/rustc_target/src/spec/armv7_unknown_trusty.rs

```diff
@@ -0,0 +1,33 @@
+use crate::spec::{Target, TargetOptions};
+
+use super::{crt_objects::LinkSelfContainedDefault, PanicStrategy, RelroLevel};
+
+pub fn target() -> Target {
+    Target {
+        // It's important we use "gnueabi" and not "musleabi" here. LLVM uses it
+        // to determine the calling convention and float ABI, and it doesn't
+        // support the "musleabi" value.
+        llvm_target: "armv7-unknown-unknown-gnueabi".into(),
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
+        arch: "arm".into(),
+        options: TargetOptions {
+            abi: "eabi".into(),
+            features: "+v7,+thumb2,+soft-float,-neon,+reserve-x18".into(),
```

Comment:
Slightly surprised to see this target enables Thumb instructions but is not named `thumbv7`.